course3/homework-string/my_string.h

```diff
@@ -1,6 +1,44 @@
 #pragma once
+#include <memory>
+#include <cstring>
 
 class MyString
 {
 	// TODO
+
+	public:
+		MyString();
+
+		MyString(const char* argString);
+
+		MyString(const MyString& argString);
+    
+        MyString(MyString&& argString);
```

Comment:
A result of using both XCode and vi, I think. In the IDE, I don't see this issue, but I will fix it.